blaze/tests/test_interactive.py

```diff
@@ -332,3 +333,23 @@ def test_scalar_sql_compute():
             dshape=dshape('var * {name: string, amount: int}'))
     d = Data(t)
     assert repr(d.amount.sum()) == '300'
+
+
+def test_no_name_for_simple_data():
+    d = Data([1, 2, 3])
+    assert repr(d) == '    \n0  1\n1  2\n2  3'
+    assert not str(d)
```

Comment:
It might be reasonable for these to have a string representation in the future (or even right now).  I suggest explicitly asserting `not d._name`